### PR TITLE
pogchamp

### PR DESCRIPTION
**Pull request by DamonOnYT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,12 @@
...
 
 ## A couple projects I'm proud of
 <!--Name - Description - My role<br>-->
-[Jinkaku](https://top.gg/bot/706271163168587848) - A discord bot for roleplayers - Owner and Lead Dev<br>
-[KeyFM](https://keyfm.net) - An online radio station - Administrator<br>
-[CloudX](https://cloudx.wtf) - An online radio station - Bot Developer<br>
-Music4All - A discord bot which played music. My first ever Discord bot. Shutdown in 2018 - Owner and Lead Dev
+[Dual](https://dual.pw) - Online Radio Station - Owner<br>
+[Securio](https://securio.tech) - A DDoS Protection and web performance company - Support Staff<br>
+[Xonos](https://xonos.host) - Game and Web hosting company - Support Staff and Developer<br>
```
Comment (HexDevv) [inline README.md:9]: i dont work here

File: README.md [hunk 2]
```diff
@@ -1,29 +1,12 @@
...
-I'm Jacques, but I'm also known as Hex or Jack. I'm a 15 year old "kinda good" Node.js developer who loves to code in their spare time.
+I'm Ben, but I'm also known as Defectiveness or benne. I'm 15 and do stupid terrible code in my spare time.
 
 ## A couple projects I'm proud of
 <!--Name - Description - My role<br>-->
-[Jinkaku](https://top.gg/bot/706271163168587848) - A discord bot for roleplayers - Owner and Lead Dev<br>
-[KeyFM](https://keyfm.net) - An online radio station - Administrator<br>
-[CloudX](https://cloudx.wtf) - An online radio station - Bot Developer<br>
-Music4All - A discord bot which played music. My first ever Discord bot. Shutdown in 2018 - Owner and Lead Dev
+[Dual](https://dual.pw) - Online Radio Station - Owner<br>
```
Comment (HexDevv) [inline README.md:7]: i dont work here

File: README.md [hunk 3]
```diff
@@ -1,29 +1,12 @@
 <h1 align="center">Hi there 👋</h1>
 
-I'm Jacques, but I'm also known as Hex or Jack. I'm a 15 year old "kinda good" Node.js developer who loves to code in their spare time.
+I'm Ben, but I'm also known as Defectiveness or benne. I'm 15 and do stupid terrible code in my spare time.
 
 ## A couple projects I'm proud of
 <!--Name - Description - My role<br>-->
-[Jinkaku](https://top.gg/bot/706271163168587848) - A discord bot for roleplayers - Owner and Lead Dev<br>
```
Comment (HexDevv) [inline README.md:7]: i work here

File: README.md [hunk 4]
```diff
@@ -1,29 +1,12 @@
...
-[KeyFM](https://keyfm.net) - An online radio station - Administrator<br>
```
Comment (HexDevv) [inline README.md:8]: i work here

File: README.md [hunk 5]
```diff
@@ -1,29 +1,12 @@
...
-[CloudX](https://cloudx.wtf) - An online radio station - Bot Developer<br>
-Music4All - A discord bot which played music. My first ever Discord bot. Shutdown in 2018 - Owner and Lead Dev
```
Comment (HexDevv) [inline README.md:10]: i work here

File: README.md [hunk 6]
```diff
@@ -1,29 +1,12 @@
...
-- 🔭 I’m currently working on ...
-- 🌱 I’m currently learning ...
-- 👯 I’m looking to collaborate on ...
-- 🤔 I’m looking for help with ...
-- 💬 Ask me about ...
-- 📫 How to reach me: ...
-- 😄 Pronouns: ...
-- ⚡ Fun fact: ...
--->
+[![Github Stats](https://github-readme-stats.vercel.app/api?username=defectivenessyt)]
```
Comment (HexDevv) [inline README.md:12]: thats not my github stats

File: README.md [hunk 7]
```diff
@@ -1,29 +1,12 @@
 <h1 align="center">Hi there 👋</h1>
 
-I'm Jacques, but I'm also known as Hex or Jack. I'm a 15 year old "kinda good" Node.js developer who loves to code in their spare time.
+I'm Ben, but I'm also known as Defectiveness or benne. I'm 15 and do stupid terrible code in my spare time.
```
Comment (HexDevv) [inline README.md:3]: im not ben

**Review by HexDevv — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (DefectivenessYT) [reply README.md:3]: but you do terrible code so

Comment (HexDevv) [reply README.md:3]: > but you do terrible code so

thats true that means im ben

**Comment by DefectivenessYT** [comment]
RIGHT BOYS I HAVE NO ACCESS TO ANYTHING ELSE BECAUSE IM IN A COMPUTER SCIENCE LESSON AND IM BORED

**Comment by DefectivenessYT** [comment]
image

**Comment by DamonOnYT** [comment]
lol f

**Comment by HexDevv** [comment]
retard

File: README.md (hunk 7 above)
Comment (DefectivenessYT) [reply README.md:3]: nigga what

**Pull request closed without merging**